scripts/convert-mapanet-to-udb.mjs

```diff
@@ -47,17 +51,17 @@ const removeRegionIfLocalityOnlyUsedOnce = (city) => {
     return city;
   }
 
-  const [locality, lastPart] = city.name.split(' (');
-  const region = lastPart.substring(0, lastPart.length - 1);
+  const [label] = city.label.split(' (');
+  const [name] = city.name.split(' (');
 
-  if (localityToTimesUsed.get(locality) > 1) {
+  if (localityToTimesUsed.get(name) > 1) {
```

Comment:
I hope the keys still match the ones inserted in `localityToTimesUsed`.

Reply:
Just checked and the JSON files before and after have the same number of entries so should be good 👍

Reply:
Nice 👍🏻